src/main/java/frc/robot/commands/LEDSwipeCommand.java

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.Random;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.LEDConstants;
+import frc.robot.subsystems.LEDSubsystem;
+
+public class LEDSwipeCommand extends CommandBase {
+  public enum SwipeType {
```

Comment:
Needs documentation describing the behavior of each different swipe type, especially for the enums with not so obvious meanings (such as kDoubleDouble)